index.html

```diff
@@ -96,19 +94,18 @@ <h1 class="descr__header">Multi-Post Stories</h1>
           <button class="see__proj__btn"><a href="">See Project</a></button>
       
         </div>
-                <div class="post__side__img">
-          <img class="post__img__mobile" src="img/Snapshoot Portfolio (1).svg" alt="">
-          
-          <img class="post__image" src="img/Snapshoot Portfolio.svg" alt="">
-        </div>
-    </div>  
-    
-    <div class="post__div">
+
         <div class="post__side__img">
           <img class="post__img__mobile" src="img/Snapshoot Portfolio (2).svg" alt="">
-          
-          <img class="post__image" src="img/Snapshoot Portfolio.svg" alt="">
+        </div>        
+
+      </div>
+      
+      <div class="post__div">
+        <div class="post__side__img">
+          <img class="post__img__mobile" src="img/Snapshoot Portfolio (3).svg" alt="">
         </div>
```

Comment:
- [ ] I  must confess your desktop design is excellent and beautiful 👏🏾 👏🏾 Well done!

But you actually traded your mobile design for the desktop. Your project in mobile view is a big mess. see below the state. It is expected that you use media queries to adapt what you already did well on mobile to desktop. I suggest that you revert your code to your last commit where your mobile view was displayed correctly thereafter you slowly use media queries to adapt it to the desktop without distorting the mobile view design. I noticed that you have a file called `desktopversion.css`, what about the `mobileversion.css`?

Your project in mobile view 👇🏾 

![image](https://github.com/sibomana123/MV__Portfolio/assets/84156714/2bea4a54-15ea-4c2a-ba4f-11430fd7a9d0)